syntax = "proto3";
package trc.serial;

message Image
{
    uint32 width = 1;
    uint32 height = 2;

    bytes pixel_data_png = 3;
}

message Texture
{
    Image image = 2;
}
